app/(admin)/admin/dashboard/navigation/page.tsx

```diff
@@ -0,0 +1,13 @@
+import { createServerComponentClient } from "@supabase/auth-helpers-nextjs";
+import { cookies } from "next/headers";
+import NavigationForm from "@/components/admin/NavigationForm";
+
+export default async function Page() {
+  const supabase = createServerComponentClient({ cookies });
+
+  const { data: navigation } = await supabase.from("SmartMenu").select("*");
+  console.log(navigation);
```

Comment:
Remove console logs and comment.

Reply:
Nebesvaik viskas cia gerai

Reply:
Everything is okay here mate

Reply:
It's not. Please keep production code clean.

Reply:
Fixed it my sugar boo